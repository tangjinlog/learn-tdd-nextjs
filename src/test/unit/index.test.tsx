import { render, screen } from '@testing-library/react';
import Home from '@/pages';

describe(`<Home />`, () => {
  it(`Home text가 보여진다.`, () => {
    const { container } = render(<Home />);

    const home = screen.getByText('Home');

    expect(home).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
