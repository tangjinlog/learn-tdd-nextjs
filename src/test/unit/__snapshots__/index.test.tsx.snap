// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Home text가 보여진다. 1`] = `
<div>
  <div>
    Home
  </div>
</div>
`;
